lupa/tests/test.py

```diff
@@ -2955,6 +2952,110 @@ def test_functions(self):
         self.testmissingref({}, lupa.as_itemgetter) # item getter protocol
         self.testmissingref({}, lupa.as_attrgetter) # attribute getter protocol
 
+################################################################################
+# tests for error conversion between Python and Lua
+
+class TestLuaErrorToPython(SetupLuaRuntimeMixin, unittest.TestCase):
+    def assertRaisesEqual(self, expected_exception, callable, *args, **kwargs):
```

Comment:
`callable` isn't the ideal name here since it shadows the `callable()` builtin function. Maybe just `func`.

Reply:
Resolved in https://github.com/scoder/lupa/pull/189/commits/c17af50a6added48703feb048e5517083f8257b0